src/Options/index.js

```diff
@@ -47,7 +47,7 @@ export interface ParseServerOptions {
   /* Disables console output
   :ENV: SILENT */
   silent: ?boolean;
-  /* The full URI to your mongodb database */
+  /* The full URI to your database. Supported databases are mongodb or postres. */
```

Comment:
postgress

Reply:
The code is auto generated, so that make sens :)